### ISSUE TITLE
It would be nice to add LaTeX support

### ISSUE THREAD

**Issue by jaijunior** [TenderOwl/Norka#58]
I use Norka daily, however I have migrated to other software because Norka does not support LaTeX for creating equations. I believe that with this addition it can become a tool that will reach greater audiences.

**Comment by amka** [comment]
Hey @jaijunior! Do you use it within markdown? I'm not a latex user thus have no idea about use cases :) I do know when LaTex is useful but could you provide me with more explanations and examples?

**Comment by jaijunior** [comment]
Yes, I use LaTeX inside markdown.I insert the LaTeX code between the $$ signs. That is:

$$
    LaTeX Code
$$
